package.json

```diff
@@ -22,31 +22,28 @@
   },
   "devDependencies": {
     "aws-sdk": "~2.2.43",
-    "babel-plugin-feature-flags": "~0.2.0",
+    "babel-plugin-feature-flags": "0.2.0",
```

Comment:
Yes that is the only change I intended to do. But I did not find a `release-2-6` branch and looking at the logs 2.6.0 release was done from this branch it seems.